beets/mediafile.py

```diff
@@ -582,7 +582,10 @@ def set(self, mutagen_file, value):
         """Set an individual value as the only value for the field using
         this style.
         """
-        self.set_list(mutagen_file, [value])
+        if value is None:
```

Comment:
That's odd—I'm a little surprised that ListStorageStyle is allowed to return None from `get`. That seems like it makes things more complicated for clients in general. Maybe that's worth fixing?